b2b-app/imports/ui/admin/register/form/form-schemas.js

```diff
@@ -90,6 +132,48 @@ const emergencyFormSchema = new SimpleSchema({
   emergencyContactMobile: SimpleSchema.RegEx.Phone,
 })
 
+const avatarFormSchema = new SimpleSchema({
+  avatar: {
+    type: String,
+    defaultValue: 'default.jpg',
+    allowedValues: [
+      'default.jpg',
+      '1.jpg',
+      '2.jpg',
+      '3.jpg',
+      '4.jpg',
+      '5.jpg',
+      '6.jpg',
+      '7.jpg',
+      '8.jpg',
+      '9.jpg',
+      '10.jpg',
+      '11.jpg',
+      '12.jpg',
+      '13.jpg',
+      '14.jpg',
+      '15.jpg',
+      '16.jpg',
+      'test11.png',
+      'test14.png',
+      'test16.png',
+      'test17.png',
+      'test18.png',
+      'test19.png',
+      'test20.png',
+      'test21.png',
+      'test24.png',
+      'test25.png',
+      'test26.png',
+      'test28.png',
+      'test29.png',
+    ],
```

Comment:
Feels like there should be a more compact way to do this, in python you could use list comprehension, not sure of the js equivalent from the top of my head. If you add a TODO comment I can condense this later though

Reply:
Yes, that crossed my mind too. Probably better still to store the file names in a database, makes adding new ones easier (no coding required)